app/app.go

```diff
@@ -258,7 +264,7 @@ func (app *App) initServices(ctx context.Context) error {
 
 	wg.Add(len(app.services))
```

Comment:
could be removed if we are no longer use goroutine

Reply:
indeed, fixed (as well as func wrapper and errors chan)